### ISSUE TITLE
add Tab Groups to output

### ISSUE THREAD

**Issue by TBugReporter** [Erisa/save-all-tab-urls#5]
Nice extension - but IMO it'd be nicer if the list somehow indicated where Tab Groups start and end along with the individual pages within each group.

**Comment by Erisa** [comment]
Good idea, I'll have to see whether browsers expose that in an API somewhere. Only reasons it hasn't happened already is because tab groups didn't really exist when it was first made and I don't use them personally.
